device_doctor/test/src/utils.dart

```diff
@@ -3,29 +3,57 @@
 // found in the LICENSE file.
 
 import 'dart:io';
+import 'dart:convert';
 
 import 'package:mockito/mockito.dart';
 import 'package:process/process.dart';
 
 class MockPlatform extends Mock implements Platform {}
 
-class MockProcessManager extends Mock implements ProcessManager {}
+class MockProcessManager extends Mock implements ProcessManager {
```

Comment:
+1 - I think a lot of Dart codebases are benefitting in creating their own mocks or writing good fakes. In the future it would be a good thing to consider here. For example, the tools team doesn't use mockito, and I've been writing fakes for app_dart which lets me validate various parts of the codebase interacting.